dev-env/bin/daml-sdk-head

```diff
@@ -97,28 +97,37 @@ echo ""
 function cleanup() {
   echo "SDK 0.0.0 failed to build/install - if you need help ask on #product-daml"
   echo "$(tput setaf 3)FAILED TO INSTALL! $(tput sgr 0)"
+
+  if [[ -n "${SDK_TEMP_DIR+x}" && -d "$SDK_TEMP_DIR" ]]; then
+    rm -rf "$SDK_TEMP_DIR"
+  fi
 }
 trap cleanup EXIT
 
 # Building here separately so the user can see the build process which could take a while
 bazel build "${BAZEL_MODE_FLAGS[@]}" //release:sdk-release-tarball.tar.gz
 
 readonly TARBALL="$(bazel info bazel-bin "${BAZEL_MODE_FLAGS[@]}")/release/sdk-release-tarball.tar.gz"
-readonly TMPDIR="$(mktemp -d)"
-mkdir -p "${TMPDIR}/sdk-head"
+readonly SDK_TEMP_DIR="$(mktemp -d)"
```

Comment:
the variable names here seem a bit confusing but I don’t have a better suggestion either.

Reply:
Agreed. If you can think of anything, let me know.